### PR TITLE
fix: exception occurred deleting files

### PR DESCRIPTION
**Pull request by Prashant-Microsoft**
## Purpose
* Exception occurred deleting files

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## How to Test
*  In the admin site if you go to delete data section, then you will get the exception

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code
```
Launch admin site
```

## What to Check
- Launch the admin site and navigate to delete data section

### CODE REVIEW

File: code/backend/batch/utilities/search/integrated_vectorization_search_handler.py [hunk 1]
```diff
@@ -48,12 +48,13 @@ def get_files(self):
     def output_results(self, results):
         files = {}
         for result in results:
-            id = result["chunk_id"]
-            filename = result["title"]
-            if filename in files:
-                files[filename].append(id)
-            else:
-                files[filename] = [id]
+            if result["chunk_id"] is not None and result["title"] is not None:
```
Comment (adamdougal) [inline code/backend/batch/utilities/search/integrated_vectorization_search_handler.py:51]: Can we add a unit test for this case?

**Review by adamdougal — CHANGES_REQUESTED** [review]
Do we know under what circumstances this can happen? Do you have a copy or a screenshot of the error? Thanks

**Comment by Prashant-Microsoft** [comment]
> Do we know under what circumstances this can happen? Do you have a copy or a screenshot of the error? Thanks

In some of the documents we are getting title field as null, that case it will give us error
image

**Comment by adamdougal** [comment]
Gotcha! My concern with this fix is that, yes it stops the exception, but it also stops the file being outputted, which means it cannot be deleted.

Could we address the issue of the title not being there instead?

**Comment by Roopan-Microsoft** [comment]
> Gotcha! My concern with this fix is that, yes it stops the exception, but it also stops the file being outputted, which means it cannot be deleted.
> 
> Could we address the issue of the title not being there instead?

Yes @adamdougal - We are also trying the same scenario to repro that issue. Will figure it out

**Comment by Prashant-Microsoft** [comment]
> Gotcha! My concern with this fix is that, yes it stops the exception, but it also stops the file being outputted, which means it cannot be deleted.
> 
> Could we address the issue of the title not being there instead?

@adamdougal Seems like this issue is intermittent, now we are not able to get the title field as null, everything working properly but still, we are working on it to repro the issue

**Comment by Prashant-Microsoft** [comment]
> Gotcha! My concern with this fix is that, yes it stops the exception, but it also stops the file being outputted, which means it cannot be deleted.
> 
> Could we address the issue of the title not being there instead?

@adamdougal Seems like this issue is intermittent, now we are not able to get the title field as null, everything working properly but still, we are working on it to repro the issue

**Comment by adamdougal** [comment]
Closing as we can't reproduce the error.

**Pull request closed without merging**